README.md

```diff
@@ -128,6 +130,16 @@ provider you like.
    GITHUB_CLIENT_ID=xxx GITHUB_CLIENT_SECRET=xxx deno run --unstable --allow-env --allow-net server.ts
    ```
 
+1. Clean-up expired KV entries as part of a cron job, if possible.
+
+   ```ts
+   import { cleanKv } from "https://deno.land/x/deno_kv_oauth@$VERSION/mod.ts";
```

Comment:
`cleanKv` might sound confusing as what's cleared are not obvious from the name. How about `cleanExpired` or similar?